realtime_ai_character/websocket_routes.py

```diff
@@ -348,8 +366,20 @@ async def stop_audio():
             # handle binary message(audio)
             elif 'bytes' in data:
                 binary_data = data['bytes']
+                # 0. Handle interim speech.
+                if speech_recognition_interim:
+                    interim_transcript: str = (await asyncio.to_thread(speech_to_text.transcribe,
+                        binary_data, platform=platform,prompt=current_speech, suppress_tokens=[0, 11, 13, 30])).strip()
```

Comment:
qq, what does suppress_tokens do?

Reply:
It's for suppressing punctuation marks`!?.,`, I did some test and found out if whisper doesn't output punctuation, the interim transcript prompt works better.

The numbers come from the output of whisper tokenizer, via [this issue](https://github.com/openai/whisper/discussions/589)